server/index.js

```diff
@@ -33,11 +34,18 @@ app.get('*', (req, res) => res.status(200).send({
 const port = parseInt(process.env.PORT, 10) || 8000;
 app.set('port', port);
 
+const createPlacements = () => {
+  if (process.env.NODE_ENV === 'development') {
+    axios.get('http://localhost:8000/api/v1/auto/place');
```

Comment:
What is this request meant to achieve?

Reply:
This is meant to add allocations data into redis database.